app/main.go

```diff
@@ -109,6 +137,25 @@ func main() {
 
 }
 
+func syncLoop(cancel context.CancelFunc, config *tsbridge.Config, ctx context.Context) {
+	func() {
```

Comment:
You shouldn't need a `func()` here, since you're already calling `syncLoop` in a goroutine.

Reply:
D'oh, fixed!